### PR TITLE
{data,tools}[GCCcore/6.4.0,intel/2017b] netCDF v4.5.0, cURL v7.56.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2101.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d9a638a3490b8bba90da66f2170a969c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2432.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b760fd4766a98d457129df86696d2094 for a full test report.

File: easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+builddependencies = [
+    ('CMake', '3.9.5'),
+    ('Doxygen', '1.8.13'),
+]
+
+# make sure both static and shared libs are built
+configopts = [
+    "-DBUILD_SHARED_LIBS=OFF ",
+    "-DBUILD_SHARED_LIBS=ON ",
```
Comment (verdurin) [inline easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2017b.eb]: I assume this is right, because it was in the previous versions. Just looks weird...

Comment (boegel) [reply easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2017b.eb]: The intention here is to build both static and shared libraries. I think they make you pick, and so if you want both, you need to do two `cmake - make - make install` cycles, which is exactly what this does: we configure once with `-DBUILD_SHARED_LIBS=OFF` (and then `make` + `make install`), and then do another cycle when configuring with `-DBUILD_SHARED_LIBS=ON`...

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**